web/src/lib/cms/previews/profile-industryspecific.tsx

```diff
@@ -42,44 +41,40 @@ const ProfilePreviewIndustrySpecific = (props: Props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [dataString]);
 
-  const { businessPersona } = getMetadataFromSlug(props.entry.toJS().slug);
-
   return (
     <ConfigContext.Provider value={{ config, setOverrides: setConfig }}>
       <div className="cms" ref={ref} style={{ margin: 40, pointerEvents: "none" }}>
         <ProfileDataContext.Provider
           value={{
             state: {
               profileData: createEmptyProfileData(),
-              flow: businessPersona || "STARTING",
```

Comment:
With this change, its only showing the preview for starting We should still be able to see the separate preview for Dakota and for Owning

Reply:
@dsueltenfuss Let's chat about this tomorrow.

Reply:
fixed